### PR TITLE
push part 3 to continue with control panel

### PR DESCRIPTION
**Pull request by RideOrDye**
Screen Shot 2021-12-12 at 4 13 04 PM

### CODE REVIEW

File: responsiveslider.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    width:50%;
+    border-style: dotted;
+    border-color: #D3D3D3;
+    border-radius: 10px;
+    height:50%;
+    background-color: lightseagreen;
+    color: blue; 
+  }
+
+  #rangeField{
```
Comment (nbeliczay) [inline responsiveslider.css]: Pls fix formatting

File: responsiveslider.html [hunk 2]
```diff
@@ -2,65 +2,44 @@
...
         <title>Parcel Sandbox</title>
         <meta charset="UTF-8" />
+        <link rel="stylesheet" href="responsiveslider.css">
     </head>
-    <!--body starts here -->
-
     <body>
         <div class="container">
+
+        <div class="panel">
```
Comment (nbeliczay) [inline responsiveslider.html]: If panel is a child of container, it should be indented one more

File: responsiveslider.html [hunk 3]
```diff
@@ -2,65 +2,44 @@
...
+                Range
+                <input type="text" placeholder="max 200" id="rangeField">
+              </label>
+            </div>
+            <button id="submitButton">submit</button>
+            </div>
+       
             <div class="content">
                 <div>
                     <ul class="nu">
```
Comment (nbeliczay) [inline responsiveslider.html:46]: missing html tags here?

File: responsiveslider.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
+
+const slider = document.querySelector(`#slider`);
+const range = document.querySelector(`.range`);
+const nu = document.querySelector(`.nu`);
+const colorPickerJs = document.getElementById("color-picker");
+const submitButton = document.querySelector("input[type=submit]");
+const buttonSize = document.getElementById('buttonSize');
+const sliderSize = document.getElementById('sliderSize');
+const submit = document.getElementById('submitButton');
```
Comment (nbeliczay) [inline responsiveslider.js]: "submit" might be a reserved word in html, pls check and rename your const.

**Comment by nbeliczay** [comment]
Pls add description to your PR and attach screenshot of config panel.

File: responsiveslider.css (hunk 1 above)
Comment (RideOrDye) [reply responsiveslider.css]: fixed

File: responsiveslider.html (hunk 2 above)
Comment (RideOrDye) [reply responsiveslider.html]: indented

File: responsiveslider.html (hunk 3 above)
Comment (RideOrDye) [reply responsiveslider.html:46]: got it

File: responsiveslider.js (hunk 4 above)
Comment (RideOrDye) [reply responsiveslider.js]: renamed submit to submitAll

**Comment by nbeliczay** [comment]
Reads good, still missing a screenshot, so can't tell what it looks like. :) 
Otherwise ready to merge!

**Comment by RideOrDye** [comment]
> Reads good, still missing a screenshot, so can't tell what it looks like. :)
> Otherwise ready to merge!

So weird, the image uploaded on my end and  all changes have been pushed.  I will look more into this. 

Screen Shot 2021-12-12 at 3 32 55 PM

**Comment by cclen** [comment]
Hi @RideOrDye , thank you for the screenshot! I see that you attached your image to the ReadME.md file. To attach a screenshot to the actual PR you can simply drag and drop an image to the description (or a comment but better in the PR description). Also, best to remove it from the ReadME.md as that is supposed to hold information that is related to the entire repo. I hope this helps to clarify why it wasn't visible in this PR😄

**Comment by RideOrDye** [comment]
Thanks @cclen. all set :)

**Pull request merged**